src/redclaw/data.clj

```diff
@@ -41,11 +47,66 @@
 (defn read-all [rs]
   (.readAll rs))
 
+(defn remove [rs v]
+  (.remove rs v))
+
+(defn contains? [rs s]
+  (.contains rs s))
+
+(defn contains-all? [rs xs]
+  (.containsAll rs xs))
+
+(defn contains-some? [rs xs]
+  (->> (some xs (read-all rs))
+       boolean))
+
+;; -----------------------------------------------------------------
+;; functions for map actions
+;; -----------------------------------------------------------------
+
 (defn put [rm k v]
   (.put rm k v))
 
 (defn put-all [rm m]
   (.putAll rm m))
 
-(defn get-all [rm]
-  (.getAll rm))
+(defn get [rm k]
+  (.get rm k))
+
+(defn get-all
+  ([rm] (get-all rm (.keySet rm)))
+  ([rm xs] (.getAll rm xs)))
```

Comment:
since this is a not one to one function `.fooBar` => `foo-bar`
it should live in a protocol
and applied to an appropriate redisson object